chromedp_test.go

```diff
@@ -116,7 +116,7 @@ func testAllocate(tb testing.TB, name string) (context.Context, context.CancelFu
 	// each test gives a huge speed-up.
 	ctx, _ := NewContext(browserCtx)
 
-	// Only navigate if we want an html file name, otherwise leave the blank page.
+	// Only navigate if we want a html file name, otherwise leave the blank page.
```

Comment:
The `h` in `html` is pronounced as `/eɪtʃ/`. So `an` is correct. https://en.wikipedia.org/wiki/HTML uses "an HTML ...".